tools/quantwiz_iq/quantwiz_iq.xml

```diff
@@ -0,0 +1,170 @@
+<tool id="quantwiz_iq" name="QuantWiz_IQ" version="2.0.1">
+    <description>Isobaric Quantitation using QuantWiz_IQ</description>
+    <requirements>
+        <requirement type="package" version="2.0">quantwiz-iq</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+cp '$input_type.input_file' '$input_type.input_file.name'
+&& cat '$new_param_file' > '$iq_params'
+#if $input_purity_file.input_purity_file_option == "history":
+&& cp '$input_purity_file.purity_type' '$input_purity_file.purity_type.name'
+&& echo "Purity_corrections_values=$input_purity_file.purity_type.name" >> '$iq_params'
+#else:
+&& PROG_PATH=`which QuantWiz_IQ`
+&& cp \${PROG_PATH%/*}/Purity_correction/$input_purity_file.purity_type .
+&& echo "Purity_corrections_values=$input_purity_file.purity_type" >> '$iq_params'
+#end if
+
+&& QuantWiz_IQ '$iq_params'
+&& cp '${input_type.input_file.name.rsplit('.',1)[0]}_${area_method}_${min_tol}.tsv' '$quant'
+&& cp '${input_type.input_file.name.rsplit('.',1)[0]}_${area_method}_${min_tol}_noquant.tsv' '$noquant'
+
+
+    ]]></command>
+    <configfiles>
+        <configfile name="new_param_file"><![CDATA[#slurp
+Spectra_Folder=.
+Input_file_extension=$input_type.file_ext
+Label_type=$input_label.label_type
+Plex=$input_label.num_plex
+min_reporters=$min_reporters
+Spectrum_area=$area_method
+Base_reporter=$base_reporter
+Normalization=$normalization
+Peak_intensity_threshold=$peak_intensity_threshold
+Tol_unit=$tol_unit
+Tol_min=$min_tol
+Tol_max=$max_tol
+#slurp
+]]></configfile>
+    </configfiles>
+    <inputs>
+        <conditional name="input_type">
```

Comment:
why is this conditional needed? `<param type="data" name="input_file" format="mgf,mzml" label="Input file"/>` should be enough.